### DISCUSSION TITLE
Request: add 'id="site-menu-button" to site menu button dropdown

### DISCUSSION THREAD

**Discussion by KatieMFritz** [craftcms/cms#8545] [Ideas]
Problem: Authors get confused about which site they are currently viewing/editing without visual differentiation. Using the Control Panel CSS plugin, we can style CP screens differently based on the current site. However, it's hard to find hooks for the current site. Some screens include the site handle in the URL, but others don't.

Solution:
With an `id` added to this element
```html
<div class="btn menubtn sitemenubtn" data-icon="world" tabindex="0" role="combobox" aria-owns="menu.Garnish528270655" aria-haspopup="true" aria-expanded="true">
  Site Name Here
</div>
```
then it would be pretty easy to use JavaScript to find the current site and apply site-specific classes.

Suggesting `site-menu-button` as the `id`, since there is already a `sitemenubtn` class on that element, but it could be anything that makes sense.

**Comment by lindseydiloreto** [comment]
Agreed! That would get us one step closer to tying up the loose ends here...

https://github.com/doublesecretagency/craft-cpbodyclasses/issues/2
